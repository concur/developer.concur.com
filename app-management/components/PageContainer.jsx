import React, { PropTypes } from 'react';

import Nav from './Nav';

const PageContainer = props => (
  <div className="container app-management">
    <Nav />
    {props.children}
  </div>
);

PageContainer.propTypes = {
  children: PropTypes.element.isRequired,
};

export default PageContainer;
