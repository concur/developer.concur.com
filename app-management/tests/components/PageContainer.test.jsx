import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import PageContainer from '../../components/PageContainer';

describe('<PageContainer />', () => {
  it('should render with a child component', () => {
    const child = <p>Test</p>;
    const page = shallow(<PageContainer>{child}</PageContainer>);

    expect(shallowToJson(page)).toMatchSnapshot();
  });
});
