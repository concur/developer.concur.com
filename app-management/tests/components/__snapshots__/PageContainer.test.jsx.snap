exports[`<PageContainer /> should render with a child component 1`] = `
<div
  className="container app-management">
  <Connect(Nav) />
  <p>
    Test
  </p>
</div>
`;
